static_src/components/app_page.jsx

```diff
@@ -113,19 +113,22 @@ export default class AppPage extends React.Component {
           </section>
           <section className={this.styler("section-card")}>
             <h3>Services</h3>
-            <p>To bind or unbind a service instance to an app view<a
+            <p>To bind or unbind a service instance to an app, follow the<a
               href="https://docs.cloud.gov/apps/managed-services/#bind-the-service-instance"
               target="_blank">
-              <span>&nbsp;</span>managed services guide</a> for more information.
+              <span>&nbsp;</span>managed services guide</a>.
```

Comment:
Hm, this is rendering without a space:

![screen shot 2016-07-08 at 11 34 57 am](https://cloud.githubusercontent.com/assets/391313/16697745/045493a0-4500-11e6-824c-a5c69f80ad00.png)